### PR TITLE
Add example and doc-test for count

### PR DESCRIPTION
**Pull request by ammkrn**
Offered in response to #7. Sorry for the confusion.

### CODE REVIEW

File: ascent/src/aggregators.rs [hunk 1]
```diff
@@ -38,6 +38,31 @@ where N: Ord + Add + Clone + Sum<N>
 }
 
 /// returns the number of tuples
+/// 
+/// # Examples
+/// 
+/// ```
+/// # use std::collections::HashSet;
+/// # use ascent_macro::ascent_run;
```
Comment (s-arash) [inline ascent/src/aggregators.rs:46]: could you change it to `use ascent::ascent_run;`?

File: ascent/src/aggregators.rs [hunk 2]
```diff
@@ -38,6 +38,31 @@ where N: Ord + Add + Clone + Sum<N>
...
+///    edge(1, 2);
+///    edge(2, 3);
+///    edge(3, 4);
+/// 
+///    num_paths(n) <-- agg n = count() in path(_, _);
+/// };
+/// // This example program is expected to produce 6 paths.
+/// let actual = res.num_paths.into_iter().collect::<HashSet<_>>(); 
+/// let expected = [(6,)].into_iter().collect::<HashSet<_>>();
+/// assert_eq!(actual, expected);
```
Comment (s-arash) [inline ascent/src/aggregators.rs:64]: I think you can simplify it to something like `assert_eq!(res.num_paths[0].0, 6)` if you find it more readable.
`num_paths` is expected to have a single fact, so this should be OK.

**Comment by ammkrn** [comment]
Thanks for the help; I implemented both of those suggestions.

**Comment by s-arash** [comment]
Great. Thanks!

**Pull request merged**